src/middleware/bearer-auth/index.ts

```diff
@@ -5,17 +5,20 @@ import { timingSafeEqual } from '../../utils/buffer'
 
 const TOKEN_STRINGS = '[A-Za-z0-9._~+/-]+=*'
 const PREFIX = 'Bearer'
+const HEADER = 'Authorization'
 
 type BearerAuthOptions =
   | {
       token: string | string[]
       realm?: string
       prefix?: string
+      header?: string
```

Comment:
I like the option name `header`, but it may be too *general*. So how about using `headerName` or something else?

Reply:
I agree. I changed it to `headerName`.